bokeh/embed.py

```diff
@@ -263,10 +274,6 @@ def file_html(models,
             using js_resources of css_resources manually.
         title (str, optional) : a title for the HTML document ``<title>`` tags or None. (default: None)
             If None, attempt to automatically find the Document title from the given plot objects.
-        js_resources (JSResources, optional): custom JS Resources (default: ``None``), if
-            resources is also provided, resources will override js_resources.
-        css_resources (CSSResources, optional): custom CSS Resources (default: ``None``), if
-            resources is also provided, resources will override css_resources.
```

Comment:
It's not documented yet, but you will be able to pass `(js_res, css_res)` tuple via `resources`.